### PR TITLE
Fix limetorrents daily

### PR DESCRIPTION
**Pull request by p0psicles**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/providers/torrent/html/limetorrents.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def __init__(self):
             'update': urljoin(self.url, '/post/updatestats.php'),
             'search': urljoin(self.url, '/search/tv/{query}/'),
             'rss': urljoin(self.url, '/browse-torrents/TV-shows/date/{page}/'),
+            'tv': urljoin(self.url, '/browse-torrents/TV-shows/'),
```
Comment (p0psicles) [inline medusa/providers/torrent/html/limetorrents.py]: @duramato why introducing tv? Are we switching from rss to tv?

File: medusa/providers/torrent/html/limetorrents.py [hunk 2]
```diff
@@ -96,7 +97,8 @@ def search(self, search_strings, age=0, ep_obj=None):
 
                     search_url = self.urls['search'].format(query=search_string)
                 else:
-                    search_url = self.urls['rss'].format(page=1)
+                    search_url = self.urls['tv']
+                    #search_url = self.urls['rss'].format(page=1)
```
Comment (p0psicles) [inline medusa/providers/torrent/html/limetorrents.py]: Why comment it here? I would just change it.

Comment (duramato) [inline medusa/providers/torrent/html/limetorrents.py]: I have hopes this is just something temporary, so i'm leaving it here to change back.

File: medusa/providers/torrent/html/limetorrents.py (hunk 1 above)
Comment (duramato) [inline medusa/providers/torrent/html/limetorrents.py]: rss was a bad variable name, it's the same the only diff is pages aren't working besides the 1st

Comment (p0psicles) [inline medusa/providers/torrent/html/limetorrents.py]: TV or rss is just a convention we use? We have lots of 'rss' URLs that work perfectly.

Comment (duramato) [inline medusa/providers/torrent/html/limetorrents.py]: `rss` here was reffering to it beeing an url for daily, just that. It isn't an xml rss or anything like that. I used `tv` becouse it is the page of latest tv shows.

**Pull request merged**